### PR TITLE
Release 04/18 -  Slash Webtasks Migrating to Node 8

### PR DESCRIPTION
**Pull request by NotMyself**
TODO:
  - [x] ask to get on the blog schedule
  - [x]  ask for art assets
  - [x] write outline
  - [x] write post
  - [x] revise intro
  - [x] add mention of permissions needed to migrate
  - [x] add section about auto migration on april 30th

screencapture-localhost-4000-blog-slash-webtasks-migrating-to-node-8-2018-04-16-10_49_59

### CODE REVIEW

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+It is that simple! Now repeat the process for each of your Node 4 based Webtasks and you will no longer have to worry about the April 30th end of life of Node 4.
+
+## What About Webtask.io
+
+Our free Serverless platform [Webtask.io](https://webtask.io/) also currently runs on Node 4. Migration to Node 8 is next on our list and is coming soon. Keep an eye on this blog for details.
+
+## Summary
+
+Migrating Slash Webtasks to Node 8 opens up a huge set of new features and capibilites. The process is fairly simple and painless. On April 30th, the Webtasks team will migrate all remaining Node 4 Webtasks to the new Node 8 envoronment.
```
Comment (glennblock) [inline _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: capabilities

Comment (glennblock) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: I'd remove the mention of auto-migrating all tasks here. The crux of the summary is you can now start developing Slash Webtasks in Node 8 and you should get started migrating your existing tasks now.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+If you encounter any issues migrating your Webtasks, please do not hesitate to contact us. You can send a support ticket or join us in our Webtask User Slack.
```
Comment (glennblock) [inline _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: You can file a support ticket on webtask.io or join us in our [slack](https://skynet.run.webtask.io/webtask-signup)

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Once you are satisfied that your webtask executes as expected in the Node 8 environment, you can use the command `/wt node8 promote {name}` to make the Node 8 version the default. This means that executing `/wt {name}` will now execute the Node 8 version of the webtask. The Node 4 version will no longer be avaiable and will be removed completely once the migration is complete.
+
+[![Promote Node 8 Webtask](https://cdn.auth0.com/website/blog/extend/slash-webtasks-migrating-to-node-8/slash-webtasks-promote-node8.png)](https://cdn.auth0.com/website/blog/extend/slash-webtasks-migrating-to-node-8/slash-webtasks-promote-node8.png)
+
+It is that simple! Now repeat the process for each of your Node 4 based Webtasks and you will no longer have to worry about the April 30th end of life of Node 4.
+
+## What About Webtask.io
+
+Our free Serverless platform [Webtask.io](https://webtask.io/) also currently runs on Node 4. Migration to Node 8 is next on our list and is coming soon. Keep an eye on this blog for details.
```
[the remark this answers is not in the document]
Comment (cfjedimaster) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Should it be serverless? I always think of Serverless as the framework.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- 2016-06-28-building-serverless-apps-with-webtask
+---
+
+Back in October of 2016 we [introduced Slash Webtasks](https://auth0.com/blog/slash-webtasks-all-your-chatops-belong-to-you/), our powerful ChatOps app for Slack. Slash Webtasks is an amazingly simple way to author custom Slack commands using Webtasks right from within Slack. The app was so popular that over 1,000,000 teams have installed it and authored over 5,000,000 Webtasks that can be called directly using the simiple `/wt task_name` interface.
+
+The underlying Webtask technology that Slash Webtasks run on is based on Node v4.9.1. The Node.js Foundation has updated their [release schedule](https://github.com/nodejs/Release#release-schedule) to show that the v4.x line is going to reach it's end of life on April 30th, 2018. That means there will be no more releases including security patches or bug fixes for the v4.x line.
+
+With the end of life in mind, the team responsible for the Webtasks technology has been working to update the platform to a Active Long Term Support version of Node.js. Today, we are happy to announce that Slash Webtasks is the first product to be migrated to Node 8.
```
[the remark this answers is not in the document]
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Removed.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+All newly created Slash Webtasks will be created in the new Node 8 environment. Teams can immediatly start taking advantage of new features and modules that rely on Node 8.
+
+On April 30th, 2018 we will automatically migrate all Slash Webtasks to the new Node 8 environment. There are some actions that you can take to proactively migrate and test your commands. The manual migration process is simple and this post will walk you through all you need to know.
+
+## Detecting Webtasks to Migrate
+
+Starting now every time a Slash Webtask command is executed on the legacy Node 4 environment, the system will display a warning message about the pending Node 4 end of life and instructions on how to migrate the task to the new Node 8 environment.
```
[the remark this answers is not in the document]
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Done.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+The underlying Webtask technology that Slash Webtasks run on is based on Node v4.9.1. The Node.js Foundation has updated their [release schedule](https://github.com/nodejs/Release#release-schedule) to show that the v4.x line is going to reach it's end of life on April 30th, 2018. That means there will be no more releases including security patches or bug fixes for the v4.x line.
+
+With the end of life in mind, the team responsible for the Webtasks technology has been working to update the platform to a Active Long Term Support version of Node.js. Today, we are happy to announce that Slash Webtasks is the first product to be migrated to Node 8.
+
+All newly created Slash Webtasks will be created in the new Node 8 environment. Teams can immediatly start taking advantage of new features and modules that rely on Node 8.
+
+On April 30th, 2018 we will automatically migrate all Slash Webtasks to the new Node 8 environment. There are some actions that you can take to proactively migrate and test your commands. The manual migration process is simple and this post will walk you through all you need to know.
```
[the remark this answers is not in the document]
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Done.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+## Detecting Webtasks to Migrate
+
+Starting now every time a Slash Webtask command is executed on the legacy Node 4 environment, the system will display a warning message about the pending Node 4 end of life and instructions on how to migrate the task to the new Node 8 environment.
+
+The webtask will continue to execute normally, the warning is to allow your team time to migrate your tasks to the new environment.
```
[the remark this answers is not in the document]
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Done.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+[![Execution Warning](https://cdn.auth0.com/website/blog/extend/slash-webtasks-migrating-to-node-8/slash-webtasks-execute-migrate.png)](https://cdn.auth0.com/website/blog/extend/slash-webtasks-migrating-to-node-8/slash-webtasks-execute-migrate.png)
+
+In this example, I have executed a StackOverflow search inside Slack. The warning message is displayed along with steps to migrate. The search results are displayed below the warning.
+
+To see a list of all webtasks your team has created in Slack, use the `/wt ls` command. A list of all webtasks will be displayed. Webtasks that need to be migrated will have a **4 emoji** next to it.
```
[the remark this answers is not in the document]
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Done.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown (hunk 2 above)
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Done.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown (hunk 1 above)
Comment (NotMyself) [reply _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: Done.

File: _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown [hunk 9]
```diff
@@ -81,6 +81,10 @@ Once you are satisfied that your webtask executes as expected in the Node 8 envi
 
 It is that simple! Now repeat the process for each of your Node 4 based Webtasks and you will no longer have to worry about the April 30th end of life of Node 4.
 
+## Migration Deadline
+
+On April 30th, Node 4 will officially reach it's end of life. On that date all remaining Slash Webtasks executing in the Node 4 environment will be automaticlly migrated. There is a chance this will cause your tasks to fail. Please migrate and test your tasks before this date.
+
 ## What About Webtask.io
```
Comment (glennblock) [inline _posts/2018-04-18-slash-webtasks-migrating-to-node-8.markdown]: very soon

**Pull request merged**